pkg/http-server/webhook-scan.go

```diff
@@ -31,6 +31,7 @@ import (
 
 // validateK8SWebhook handles the incoming validating admission webhook from kubernetes API server
 func (g *APIHandler) validateK8SWebhook(w http.ResponseWriter, r *http.Request) {
+	zap.S().Info("handle: validating webhook request")
```

Comment:
I honestly feel this should be logged in debug mode level and not in info

Reply:
Also, for logging in the incoming requests, we should work on https://github.com/accurics/terrascan/issues/784

Reply:
it is an informational log and hence to be logged in info.
Also, I had a discussion with Yusuf, any other logging related changes can be taken up with #784.